SwiftUIHW/SwiftUIHW/PlayerViewModel.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  PlayerViewModel.swift
+//  SwiftUIHW
+//
+//  Created by Daniil on 18.01.2023.
+//
+
+import SwiftUI
+import AVFoundation
+
+/// Вью модель плеера
+final class PlayerViewModel: ObservableObject {
+    // MARK: - Private properties
+    
+    @Published public var maxDuration: Float = Constants.floatDefaultValue
```

Comment:
```suggestion
    @Published var maxDuration: Float = Constants.floatDefaultValue
```
не пиши нигде public